rockcraft/commands/lifecycle.py

```diff
@@ -46,3 +46,40 @@ class PackCommand(_LifecycleCommand):
     def run(self, parsed_args):
         """Run the command."""
         lifecycle.pack()
+
+
+class InitCommand(_LifecycleCommand):
+    """Initialize a rockcraft project."""
+
+    name = "init"
+    help_msg = "Initialize a rockcraft project"
+    overview = textwrap.dedent(
+        """
+        Initialize a rockcraft project by creating a minimalist,
+        yet functional, rockcraft.yaml file in the current directory.
+        """
+    )
+
+    _INIT_TEMPLATE_YAML = textwrap.dedent(
+        """\
+            name: my-rock-name # the name of your ROCK
+            base: ubuntu:20.04 # the base environment for this ROCK
+            version: '0.1' # just for humans. Semantic versioning is recommnended
+            summary: Single-line elevator pitch for your amazing ROCK # 79 char long summary
+            description: |
+                This is my my-rock-name's description. You have a paragraph or two to tell the
+                most important story about it. Keep it under 100 words though,
+                we live in tweetspace and your description wants to look good in the
+                container registries out there.
+            license: GPL-3.0 # your application's SPDX license
+
+            parts:
```

Comment:
with pebble being a default part we probably no longer need this, which is for the better!

Reply:
no, you still need it.

`parts` is a mandatory field. The FieldValidation will fail if you don't pass it. Adding pebble as an implicit part happens after the YAML has been loaded. So `parts` must exist before that

Reply:
Yes, this can be done entry for pebble is added before we actually process the entry